src/btree/col_srch.c

```diff
@@ -127,7 +127,7 @@ __wt_col_search(
 
     /* Search the internal pages of the tree. */
     current = &btree->root;
-    for (depth = 2, pindex = NULL;; ++depth) {
+    for (depth = 1, pindex = NULL;; ++depth) {
```

Comment:
nit: instead of a hardcoded value, can we make it a macro with a name please?

something like `#define MIN_BTREE_DEPTH 1` or something more descriptive

Reply:
@tdlinh2712. The `depth` variable is just a counter. I'm not sure it increases clarity to have a macro to say "we count starting from one". 

I've expanded the comment before the loop to explain more clearly what is going on here. See [d088618](https://github.com/wiredtiger/wiredtiger/pull/11979/commits/d0886183925558088dedd6d6e1e7cc35199b7eeb)